utility.css

```diff
@@ -36,11 +36,48 @@
 
 /* code snippet class */
 .code-snipet-container {
-  box-sizing: border-box;
   width: 100%;
-  max-height: 300px;
+  max-height: 450px;
   padding: 1rem 0;
-  background-color: #fafafa;
-  margin: 2rem 0;
```

Comment:
why  you use margin property two times , is there any reason ?

Reply:
these are the deleted props from css file